src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java

```diff
@@ -328,41 +328,68 @@ public static List<OcrDataField> getC63Data() {
 
     public static List<OcrDataField> getA60Data() {
         List<OcrDataField> fieldList = new ArrayList<>();
+
         OcrDataField ocrDataFirstNameField = new OcrDataField();
         ocrDataFirstNameField.setName("applicant1_firstName");
-        ocrDataFirstNameField.setValue("firstName");
+        ocrDataFirstNameField.setValue("applicant1_firstName");
         fieldList.add(ocrDataFirstNameField);
 
         OcrDataField ocrDataLastNameField = new OcrDataField();
         ocrDataLastNameField.setName("applicant1_lastName");
-        ocrDataLastNameField.setValue("LastName");
+        ocrDataLastNameField.setValue("applicant1_lastName");
```

Comment:
better to use constant. applicable everywhere.

Reply:
Constants should have been created by previous changes. Many constants already created when reused.

Reply:
We have to be extremely careful introducing constants in this file as testdata affects many unrelated changes to this ticket